pulp_container/app/registry.py

```diff
@@ -62,6 +62,10 @@ async def _dispatch(file, headers):
             The :class:`aiohttp.web.FileResponse` for the file.
 
         """
+        path = os.path.join(settings.MEDIA_ROOT, file.name)
+        if not os.path.exists(path):
+            raise PathNotResolved(path)
```

Comment:
Should this be guarded to the case where we use the default file storage?
os.path.exists will never succeed if the data is in s3, no?

Reply:
in case s3 is used than we will use s3redirects, this code path is not executed.